src/views/registration-js/content.php

```diff
@@ -125,7 +135,12 @@ class="tribe-tickets__item__attendee__fields__form <?php echo sanitize_html_clas
 		</div>
 	</div>
 	<div class="tribe-tickets__registration__footer">
-		<p class="tribe-tickets-notice tribe-tickets-notice--non-ar">
+		<p
+			class="tribe-tickets-notice tribe-tickets-notice--non-ar"
+			<?php if ( empty( $non_meta_count ) ) : ?>
+				style="display: none;"
```

Comment:
I thought inline `style="display:none"` was not recommended and we should be using a class instead, is that right?

Reply:
See comment above: https://github.com/moderntribe/event-tickets/pull/1382/files#r330655751